### ISSUE TITLE
Typo - Initial version code recommended is inconsistent (Major and Minor versus Major, Minor, and Patch?) - installable-code.md

### ISSUE THREAD

**Issue by pb-413** [pyOpenSci/python-package-guide#257]
During the tutorial, changing the initial version in the `project.toml` to `"0.1.0"` is recommended; then, in the following example, it is listed as only `"0.1"`.

https://github.com/pyOpenSci/python-package-guide/blob/7b422d4256cdb89d40c8b2b19227e4a5d8078817/tutorials/installable-code.md?plain=1#L393C1-L400C28

I'm a fan of Semantic Versioning, but depending on your audience and intent, consider standardizing this version representation.

Happy to take up this work; just need to know which way to lean.

**Comment by ucodery** [comment]
I agree this inconsistency should be resolved. I don't care too much which way but lean toward `"0.1"` as it is the simpler string and this is a beginner guide. To packaging tools these two versions are exactly equivilant.

**Comment by pb-413** [comment]
Filed PR: #262

**Comment by ucodery** [comment]
I greatly appreciate the PR, but we still have 0.1.0s in the docs elsewhere. Opening until those are fixed.

It would be nice to also lint against us doing this again in the future. That's probably a separate issue.

**Comment by pb-413** [comment]
Understood! I'll take another pass over the docs.

Sorry about the eagerness!
